frontend-pages/profile-page.php

```diff
@@ -36,11 +36,14 @@ function ssv_profile_page_setup($content)
 
     if (isset($_GET['view']) && $_GET['view'] == 'pdf') {
         require_once(ABSPATH . 'wp-content/plugins/ssv-frontend-members/include/fpdf/SSV_FPDF.php');
+        if (isset($_GET['user_id'])) {
+            $frontendMember = FrontendMember::get_by_id($_GET['user_id']);
+        } else {
+            $frontendMember = FrontendMember::get_current_user();
+        }
         $pdf = new SSV_FPDF();
-        $pdf->AddPage();
-        $pdf->SetFont('Arial','B',16);
-        $pdf->Cell(40,10,'Hello World!');
-        $pdf->Output();
+        $pdf->build($frontendMember);
+        $pdf->Output('F', ABSPATH . 'wp-content/plugins/ssv-frontend-members/exported-profiles/' . str_replace(' ', '_', $frontendMember->display_name) . '.pdf');
```

Comment:
It doesn't need to be with GET, it can also be in a session or something, but I prefer GET but I prefer it over the other options. And WordPress isn't **that** bad (it is the security that prevents this).